components/nds-callout/scss/callout.scss

```diff
@@ -0,0 +1,38 @@
+////
+/// @group components
+////
+
+@import '~@nice-digital/nds-core/scss/core';
+
+.callout {
+  background: $nds-colour-nice-white;
+  height: auto;
+  margin-bottom: rem($nds-spacing-large);
+  &__image img {
+    width: 100%;
+  }
+  &__body {
+    padding: 1rem;
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      margin-top: 0;
```

Comment:
why don't we do a `> :first:child` with no margin top and a `> :last:child` with no margin-bottom like we do with panel, and then we don't need the list of headings